core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-row/template-builder-row.component.scss

```
@@ -0,0 +1,43 @@
+@use "variables" as *;
+:host {
+    display: flex;
+    height: 16rem;
+    border: 1px solid $color-palette-gray-300;
+    border-radius: $border-radius-md;
+}
+
+.row__drag-container,
+.row__actions-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 1.1875rem;
```

Use a `spacing` variable for `padding`, `margin`, and `gap`.
```suggestion
    padding: 0 $spacing-3;
```